sdk/nodejs/runtime/settings.ts

```diff
@@ -171,6 +178,25 @@ export function getMonitor(): Object | undefined {
     return monitor;
 }
 
+/** @internal */
+export interface SyncInvokes {
+    requests: number;
+    responses: number;
+}
+
+let syncInvokes: SyncInvokes | undefined;
+
+/** @internal */
+export function tryGetSyncInvokes(): SyncInvokes | undefined {
+    if (!syncInvokes && options.syncDir) {
```

Comment:
nit: `syncInvokes === undefined` if that's the only condition we expect.

Reply:
Sure.  Though `!` is fairly idiomatic for js/ts and our codebase.  but i don't mind being totally explicit.